docs/reference/sql/functions/overview.md

```diff
@@ -302,6 +302,30 @@ SELECT json_is_object(parse_json('1'));
 +---------------------------------------+
 ```
 
+* `json_path_exists(json, path)` to check whether a path exists in a JSON value.
+
+If the path is invalid, the function will return an error.
+
+If the path or the JSON value is NULL, the function will return a NULL value.
+
+```sql
+SELECT json_path_exists(parse_json('{"a": 1, "b": 2}'), 'a');
+
++------------------------------------------------------------------+
+| json_path_exists(parse_json(Utf8("{"a": 1, "b": 2}")),Utf8("a")) |
++------------------------------------------------------------------+
+|                                                                1 |
++------------------------------------------------------------------+
```

Comment:
```suggestion
+------------------------------------------------------------------+
|                                                            false |
+------------------------------------------------------------------+
```

Reply:
0/1 is consistent to previous functions in the docs, do we need to change them all?

https://github.com/GreptimeTeam/docs/blob/24b71920caae49d4d3b45823bea838ee4367dbd6/docs/reference/sql/functions/overview.md?plain=1#L189-L198

Reply:
0/1 is consistent with previous docs, do we need to change them all?
https://github.com/GreptimeTeam/docs/blob/24b71920caae49d4d3b45823bea838ee4367dbd6/docs/reference/sql/functions/overview.md?plain=1#L189-L198

Reply:
> docs/docs/reference/sql/functions/overview.md

Oh, I see.